packages/modular-scripts/src/program.ts

```diff
@@ -84,22 +84,64 @@ program
   )
   .option('--verbose', 'Run yarn commands with --verbose set')
   .option('--private', 'Enable the building of private packages', false)
+  .option(
+    '--changed',
+    'Build only for workspaces that have changed compared to the branch specified in --compareBranch',
+    false,
+  )
+  .option(
+    '--descendants',
+    'Additionally build workspaces that the specified packages directly or indirectly depend on (can be combined with --changed)',
+    false,
+  )
+  .option(
+    '--ancestors',
+    'Additionally build workspaces that are directly or indirectly depended on by the specified packages (can be combined with --changed)',
```

Comment:
Bit nitpicky but:
"Additionally build workspaces that directly or indirectly depend on the specified packages (can be combined with --changed)"
would be clearer - 'depended on by'  had me confused

Reply:
Yeah, you're right, what I wrote was incorrect. Fixed.